libs/prelude/Prelude/Interfaces.idr

```diff
@@ -174,12 +174,24 @@ interface Applicative m => Monad m where
   (>>=) x f = join (f <$> x)
   join x = x >>= id
 
-%allow_overloads (>>=)
+public export
+(=<<) : Monad m => (a -> m b) -> m a -> m b
```

Comment:
This one is already defined at: contrib/Control/Monad/Syntax.idr

Reply:
Thanks for the heads up, added `<<` to the same Control.Monad.Syntax file and removed both flipped additions on prelude but moved this to #856 to separate the despecialization from the concurrency bit.